src/helpers.ts

```diff
@@ -17,6 +17,9 @@ export const findLatestDraft = (documentId: string, ignoreI18n = true) => {
       (docs: SanityDocument[]) =>
         docs.find(doc => doc._id.includes('draft')) ?? docs[0]
     )
+    .catch(err => {
+      throw err
+    })
```

Comment:
I'm not sure if you need to catch the error and throw it... I know that if we are using `fetch` function or other promises functions, if we don't declare the catch function, it will throw the error. I did this example to test: https://codesandbox.io/s/affectionate-kapitsa-g0ih7?file=/src/index.js

Reply:
Does that apply to other instances you see, or just this one? I did a lot of trial and error to see which ones were needed and there were some cases where the error wouldn't make it all the way back to the `importFile()` to be caught and thrown in the translations tab plugin (the other repo that uses this one) unless I caught the error, but there were so many places that I may have left some that I didn't need to.

Reply:
It should apply to other instances too... Basically you can remove any code like this:
```
.catch(err => {
  throw err
})
```
and replace any code like this:
```
async function () {
  try {
    // code
  } catch (e) {
    throw e
  }
}

// TO

async function () {
  // code
}
```

Reply:
Okay thanks! I'll try removing them and make sure I'm still getting the errors at the end.

Reply:
Thanks so much @sanches89! You were totally right, the errors made it through when I removed the `catch`es that weren't doing anything extra. Could have sworn they fixed it at one point, but it must have been the other changes that sent the errors through (likely in `getTranslation` where I added a thrown error if no response body is turned).